test/test_helper.rb

```diff
@@ -4,15 +4,12 @@
 
 module ActiveSupport
   class TestCase
+    include Devise::Test::IntegrationHelpers
+
     # Run tests in parallel with specified workers
     parallelize(workers: :number_of_processors)
 
     # Setup all fixtures in test/fixtures/*.yml for all tests in alphabetical order.
     fixtures :all
-
-    # Add more helper methods to be used by all tests here...
-    def sign_in(user)
-      post sessions_path, params: {email: user.email, password: "password"}
```

Comment:
not a valid path anymore, I have removed the method and used the one provided by the Devise integration helper